lib/Chat/Parser/SystemMessage.php

```diff
@@ -817,6 +817,10 @@ protected function getFileFromShare(Room $room, ?Participant $participant, strin
 					$data['width'] = (string) $sizeMetadata['width'];
 					$data['height'] = (string) $sizeMetadata['height'];
 				}
+
+				if (isset($sizeMetadata['blurhash'])) {
+					$data['blurhash'] = $sizeMetadata['blurhash'];
```

Comment:
Hmm we should add it to https://github.com/nextcloud/server/blob/master/lib/public/RichObjectStrings/Definitions.php#L294 and https://github.com/nextcloud/spreed/blob/main/lib/ResponseDefinitions.php#L93 I guess